src/conf.c

```diff
@@ -77,23 +78,29 @@ int conf_loadfile( conf_t *conf, char *file )
 		fscanf( fp, "%100[^\n#]s", s );
 		fscanf( fp, "%*[^\n]s" );
 		fgetc( fp );			/* Skip newline */
-		if( strchr( s, '=' ) == NULL )
+		tmp = strchr( s, '=' );
+		if( tmp == NULL )
 			continue;		/* Probably empty? */
+
 		sscanf( s, "%[^= \t]s", key );
+
+		/* skip first char as it is the '=' */
+		while( *( ++ tmp ) )
```

Comment:
I can think of the following improvement to the loop:

	/* Skip the "=" and any spaces following it */ 
	while( isspace( * ++ tmp ) ); /* XXX isspace(0) is false */
	value = tmp;
	while ( *tmp && !isspace( *tmp ) ) tmp++;
	*tmp = 0;